includes/html/common/availability-map.inc.php

```diff
@@ -217,11 +218,13 @@
                     $deviceLabelOld = 'availability-map-oldview-box-up';
                     $host_up_count++;
                 }
+                $updowntime = "" . ($device['uptime'] ? " - " : "") . formatUptime($device['uptime']);
```

Comment:
wouldn't it be better to keep the GUI styling `($device['uptime'] ? " - " : "") .`  on to the link so `$updowntime` really only contains uptime/downtime?

Reply:
no. It should also contain the name of the device